scripts/init.sql

```diff
@@ -0,0 +1,30 @@
+create table gift_certificate
+(
+    id bigint auto_increment,
+    name varchar(50),
+    description varchar(50),
```

Comment:
I believe that the field description should be bigger than 50 symbols.

Reply:
To be honest  in real database I use it's actually has 100 symbols. Fixed